<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CE5CreateTablesForHall</class>
 <widget class="QDialog" name="CE5CreateTablesForHall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create tables for hall</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="C5LineEditWithSelector" name="leHallName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Hall</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="C5LineEditWithSelector" name="leHall">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Prefix</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5LineEditWithSelector" name="lePrefix">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>First</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEditWithSelector" name="leStart">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="C5LineEditWithSelector" name="leCount">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Count</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>122</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCreate">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>121</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnNewHall">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new.png</normaloff>:/new.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
